package.json

```diff
@@ -3,11 +3,14 @@
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "ng serve --host 0.0.0.0",
+    "dev": "ng serve --host 0.0.0.0 --disable-host-check",
```

Comment:
--host 0.0.0.0 and --disable-host-check is needed so that I can view localhost:4200 that is served on one of my Mac, view it on my other Mac within the same network. Essentially bypassing the safeguarding feature implemented by Webpack.